examples/ethereum/rosetta-cli-conf/mainnet/config.json

```diff
@@ -3,13 +3,47 @@
     "blockchain": "Ethereum",
     "network": "Mainnet"
   },
+  "online_url": "http://localhost:8080",
```

Comment:
Now I think we should create a rosetta-cli-conf/devnet, instead of messing up the mainnet / testnet ones